File: gradevaluator.js
let mark = '90'

if (mark > 90 && mark <= 100) {
    console.log("Grade A");
} else if (mark > 80 && mark <= 90) {
    console.log("Grade B");
} else if (mark > 70 && mark <= 80) {
    console.log("Grade C");
} else if (mark > 60 && mark <= 70) {
    console.log("Grade D");
} else if (mark >= 0 && mark <= 60) {
    console.log("Grade F");
} else {
    console.log("Invalid marks! Enter between 0 and 100");
}

File: gradevaluator.py
mark = float(input("Enter the marks: "))
if mark > 90 and mark <= 100:
    print("Grade A")
elif mark > 80 and mark <= 90:
    print("Grade B")
elif mark > 70 and mark <= 80:
    print("Grade C")
elif mark > 60 and mark <= 70:
    print("Grade D")
elif mark >= 0 and mark <= 60:
    print("Grade F")
else:
    print("Invalid marks! Enter between 0 and 100") 